docs/docs/01-welcome/01-index.md

```diff
@@ -4,37 +4,34 @@ slug: /
 
-## What is Ignite CLI?
+## Key Features of Ignite CLI
 
-Ignite CLI is an easy-to-use CLI tool for creating and maintaining sovereign application-specific blockchains.
-Blockchains created with Ignite CLI use Cosmos SDK and Tendermint. Ignite CLI and the Cosmos SDK modules are written in
-the Go programming language. The scaffolded blockchain that is created with Ignite CLI includes a command line interface
-that lets you manage keys, create validators, and send tokens.
-
-With just a few commands, you can use Ignite CLI to:
-
-- Create a modular blockchain written in Go
-- Scaffold modules, messages, types with CRUD operations, IBC packets, and more
-- Start a blockchain node in development with live reloading
-- Connect to other blockchains with a built-in IBC relayer
-- Use generated TypeScript/Vuex clients to interact with your blockchain
-- Use the Vue.js web app template with a set of components and Vuex modules
+- **Simplified Blockchain Development:** Ignite CLI, leveraging Cosmos SDK, makes building sovereign application-specific blockchains intuitive and efficient.
+- **Comprehensive Scaffolding:** Easily scaffold modules, messages, CRUD operations, IBC packets, and more, expediting the development of complex functionalities.
+- **Development with Live Reloading:** Start and test your blockchain node with real-time updates, enhancing your development workflow.
+- **Frontend Flexibility:** Utilize pre-built templates for Vue.js, React, Typescript or Go, catering to diverse frontend development needs.
+- **Inter-Blockchain Communication (IBC):** Seamlessly connect and interact with other blockchains using an integrated IBC relayer, a key feature of the Cosmos SDK.
+- **CometBFT Integration:** Built with the recently rebranded CometBFT (formerly Tendermint), ensuring robust consensus mechanisms in your blockchain solutions.
+- **Cross-Domain Applications:** Ideal for developing a wide range of blockchain applications in sectors like DeFi, NFTs, supply chain management, smart contracts (EVM or WASM), and decentralized exchanges (DEXes).
 
 ## Install Ignite CLI
 
-To install the `ignite` binary in `/usr/local/bin` run the following command:
+Get started with Ignite CLI by running this simple installation command:
 
 ```
 curl https://get.ignite.com/cli! | bash
 ```
 
+## Embracing the Cosmos Ecosystem
+
+Ignite CLI is your entry point into the vibrant Cosmos ecosystem, a hub of innovation where you can explore a range of applications, from wallets and explorers to smart contracts and DEXes, all powered by CometBFT and the Cosmos SDK.
+This ecosystem is home to over [$50 billion worth of blockchain projects](https://cosmos.network/ecosystem/tokens/), showcasing the scalability and versatility of the technologies at play.
```

Comment:
```suggestion
Ignite is a cornerstone in the Cosmos ecosystem, fostering the development of some of the world's most valuable chains.
```

Reply:
I like the former more. It's convincing that the Cosmos ecosystem secures more than 50B of funds.